gpkit/constraints/model.py

```diff
@@ -82,6 +82,8 @@ def __init__(self, cost=None, constraints=None, *args, **kwargs):
     solve = _solve_fctry(_progify_fctry(GeometricProgram, "solve"))
     localsolve = _solve_fctry(_progify_fctry(SequentialGeometricProgram,
                                              "localsolve"))
+    penalty_ccp_solve = _solve_fctry(_progify_fctry(SequentialGeometricProgram,
+                                                    "penalty_ccp_solve"))
```

Comment:
(whistles appreciatively)

Reply:
Still don't fully understand what this does, all I know is that it allows me to call it on a Model as well as a SGP :p